nDPI-patch/README.md

```diff
@@ -196,3 +196,33 @@ to
 #define assert(x)
 #endif
 ```
+
+### Kernel Version Mismatch
+
+Between versions 5.4 and 5.8, the Linux Kernel changed how it keeps
```

Comment:
"Kernel" isn't a proper noun and doesn't need to be capitalized here.

Reply:
Good catch.